zk_toolbox/crates/zk_inception/src/commands/external_node/args/run.rs

```diff
@@ -0,0 +1,15 @@
+use clap::Parser;
+use serde::{Deserialize, Serialize};
+
+use crate::messages::{MSG_SERVER_ADDITIONAL_ARGS_HELP, MSG_SERVER_COMPONENTS_HELP};
+
+#[derive(Debug, Serialize, Deserialize, Parser)]
+pub struct RunExternalNodeArgs {
+    #[clap(long)]
+    pub reinit: bool,
+    #[clap(long, help = MSG_SERVER_COMPONENTS_HELP)]
+    pub components: Option<Vec<String>>,
```

Comment:
Could use `enum` and `clap::ValueEnum`

Reply:
I'm not sure we have to have this dependency. we are adding components back and force constantly